src/xtgeo/cube/_cube_import.py

```diff
@@ -364,6 +364,17 @@ def import_xtgregcube(mfile, values=True):
 
     results = {myattr: req[myattr] for myattr in reqattrs}
 
+    # For backwards compatability, xtgeo outputs files with the undef field set
+    # although we do not support initializing with any other value.
+    # As xtgeo-format is only written/read by xtgeo as far as we know, this should
+    # be unproblematic for now.
+    if "undef" in results:
```

Comment:
Perhaps
`undef = results.pop("undef", None)
`
to reduce levels?

Reply:
Done